docs/releases/v1.6.0.md

```diff
@@ -0,0 +1,370 @@
+# Release notes
+
+This release includes package updates for nginx and dual-nginx, forecastle and cert-manager.
+
+## Changelog
+
+Changes between `1.5.0` and this release: `1.6.0`
+
+- Updated forecastle from     to    
+- Update nginx, nginx-gke and dual-nginx from    to    
+- Update cert-manager from 0.9.1 to 0.14.1
+
+
+## Upgrade path
+
+### forecastle
+
+No further action. Just apply this new kustomize project.
+
+```bash
+$ kustomize build katalog/forecastle | kubectl apply -f -
+```
+
+### nginx, nginx-gke and dual-nginx
+
+No further action. Just apply this new kustomize project.
+
+```bash
+$ kustomize build katalog/nginx | kubectl apply -f -
+$ kustomize build katalog/dual-nginx | kubectl apply -f -
+```
+
+### cert-manager
+
+This procedure was designed to apply the update at once. Jetstack team have written the upgrade path for every
```

Comment:
Why is not a script? it need the effort to automated it (IMHO if we're going to migrate more than 3 times it worth it)

Reply:
Maybe we can automate somehow some parts. 
But for sure we should not automate everything. *(cert-manager provides a tool to automate some parts but not all).*

I don't know... IMHO, this (cert-manager) is a "critical" part in a lot of cluster and we don't have enough capacity/time/resources to build a super awesome/robust-enough migration tool on every release (remember that cert-manager breaks compatibility "almost" every release)

So, i prefer to write a "run-book" that the operator should understand carefully. If the procedure needs more info, let me know!

Reply:
Thank you for explaining it, I´m just unaware of the responsabilities/capabilities of fury so. I'll love to see it automated somehow, but I understand the nuances.